assets/css/styles.css

```diff
@@ -557,3 +558,126 @@ button {
     background-size: 100%;
   }
 }
+
+#pop-section {
+  position: fixed;
+  height: 100vh;
+  display: none;
+  border-radius: 8px;
+  width: 100%;
+  top: 0;
+  background: #fff;
+  flex-direction: column;
+  align-items: center;
+}
+
+#pop-section .close-wrapper {
+  width: 94%;
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  margin-top: 1em;
+}
+
+#pop-section .works-card {
+  height: 90%;
+  width: 94%;
+  margin: 1em 1em 3em;
+}
+
+#pop-section p {
+  font-size: 16px;
+  line-height: 30px;
+  width: 100%;
+  text-align: left;
+}
+
+#pop-section .title-container {
+  display: flex;
+  flex-direction: row;
+  width: 70%;
+  justify-content: space-between;
+}
+
+#pop-section h4 {
+  font-size: 32px;
+  line-height: 44px;
+  width: 100%;
+  color: #172b4d;
+  text-align: left;
+}
+
+#pop-section button {
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.works-card-inner-pop {
+  display: flex;
+  flex-direction: row;
+  border-radius: 8px;
+  padding: 16px 0;
+  width: 100%;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
```

Comment:
- [ ] [REQUIRED] Please fix the mistakes in the mobile version design, I have shown the difference between the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) and your design in the picture below!

![image](https://user-images.githubusercontent.com/63915024/189192103-23c48772-d78b-4f7a-8ea6-906207290b7a.png)